### PR TITLE
[DROOLS-3167] Adjust Index/Description columns width.

### PR DESCRIPTION
**Pull request by gitgabrio**
@kkufova @Rikkola @danielezonca 
Scope of this PR is to reduce the width of the "Index" column and move the gained space to the "Details" one.

### CODE REVIEW

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/utils/ScenarioSimulationUtils.java [hunk 1]
```diff
@@ -232,6 +233,17 @@ public static ScenarioGridColumn getScenarioGridColumn(ScenarioSimulationBuilder
         return headerBuilder;
     }
 
+    protected static double getColumnWidth(String columnId) {
+        switch (columnId) {
+            case "Index":
```
Comment (danielezonca) [inline drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/utils/ScenarioSimulationUtils.java]: You can use ExpressionIdentifier.INDEX.getName()
WDYT?

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/utils/ScenarioSimulationUtils.java [hunk 2]
```diff
@@ -232,6 +233,17 @@ public static ScenarioGridColumn getScenarioGridColumn(ScenarioSimulationBuilder
...
+                return 70;
+            case "Description":
```
Comment (danielezonca) [inline drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/utils/ScenarioSimulationUtils.java]: You can use ExpressionIdentifier.DESCRIPTION.getName()
WDYT?

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/utils/ScenarioSimulationUtils.java (hunk 1 above)
Comment (gitgabrio) [reply drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/utils/ScenarioSimulationUtils.java]: I had to create an enum, but I think it was somehow missing anyway, so better that way. It is a little bit more verbose but less "magic words" around

**Comment by gitgabrio** [comment]
@manstis 
Hi! Could you please merge this, squashing commits? Thx!

**Pull request merged**